.github/workflows/ci.yml

```
@@ -448,7 +448,7 @@ jobs:
       - name: Run Sentry self-hosted e2e CI
         # Skip if a fork as they cannot upload to ghcr and this test attempts to pull
         # the image from ghcr
```

Same suggestion as above.
```suggestion
        # Skip if they cannot upload to ghcr (e.g. a fork) and this test attempts to pull
        # the image from ghcr
```